### PR TITLE
Add notifier template for apiv2, used for reset password process

### PR DESCRIPTION
**Pull request by gdchamal**
Related to https://github.com/CaliOpen/Caliopen/pull/634

### CODE REVIEW

File: roles/apiv2/templates/caliopen-api.yaml.j2 [hunk 1]
```diff
@@ -35,7 +35,10 @@ ApiConfig:
     host: {{ hostvars['cache1']['backend_ip'] }}:6379
     password: ""      #no password set
     db: 0             #use default db
-  adminUsername: admin  # username on whose behalf notifiers will act. This admin user must have been created before by other means.
+  NotifierConfig:
+    base_url: https://{{ caliopen_domain_name }}
+    admin_username: admin
+    templates_path: "/etc/caliopen/notifiers/templates"
```
Comment (sapiens-sapide) [inline roles/apiv2/templates/caliopen-api.yaml.j2]: need trailing slash for templates_path

File: roles/apiv2/templates/caliopen-api.yaml.j2 [hunk 2]
```diff
@@ -35,7 +35,10 @@ ApiConfig:
     host: {{ hostvars['cache1']['backend_ip'] }}:6379
     password: ""      #no password set
     db: 0             #use default db
-  adminUsername: admin  # username on whose behalf notifiers will act. This admin user must have been created before by other means.
+  NotifierConfig:
+    base_url: https://{{ caliopen_domain_name }}
```
Comment (sapiens-sapide) [inline roles/apiv2/templates/caliopen-api.yaml.j2:39]: does `{{ caliopen_domain_name }}` resolve without a trailing slash ?

Comment (gdchamal) [reply roles/apiv2/templates/caliopen-api.yaml.j2:39]: no it's a FQDN not an URL

File: roles/apiv2/templates/caliopen-api.yaml.j2 (hunk 1 above)
Comment (gdchamal) [reply roles/apiv2/templates/caliopen-api.yaml.j2]: done in #b5073f1

**Pull request merged**